.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ jobs:
             . venv/bin/activate
             mkdir test-reports
             pip freeze > test-reports/pip_versions.txt
-            pytest --cov=ml-agents --cov=ml-agents-envs --cov=gym-unity --cov-report html --junitxml=test-reports/junit.xml -p no:warnings
+            pytest -n 2 --cov=ml-agents --cov=ml-agents-envs --cov=gym-unity --cov-report html --junitxml=test-reports/junit.xml -p no:warnings
```

Comment:
Using "-n auto" runs a lot of workers, which crash on CI. We're currently running with 2 vCPU, so hardcode this for now (it's probably getting the number of physical cores)